.changeset/lucky-dolphins-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+"@espressive/cascara": patch
+---
+
+chore(FDS-401): [Danger.js] - Validate if target branch requires release branch
```

Comment:
@rociobaena this is a tooling change, not a change to code. It should not have a Changeset.

Reply:
Yes, you're right. It does not make sense. I'll remove it.